Community, podcast, and debug screen design updates

cdanwards opened the pull request:
## Please verify the following:

- [x] `yarn ci:test` **jest** tests pass with new tests, if relevant
- [x] `README.md` has been updated with your changes, if relevant

## Describe your PR

### This PR is some of the design feedback that Jenna had after fleshing some more of the screens out.


| Trello Ticket & Description | Image from App |
| ----------- | ----------- |
| [Logo size for brands and community page](https://trello.com/c/2pbjUJjv/18-logo-size-for-brands-on-community-page) | <img width="211" alt="Screen Shot 2022-09-13 at 11 16 10 AM" src="https://user-images.githubusercontent.com/8878532/190496489-835bd029-a593-4326-8bcc-9bd7f6ff99ae.png"> |
| [Divider lines for list items](https://trello.com/c/ECV3R8eE/17-divider-lines-for-list-items) | <img width="211" alt="Screenshot of divider lines being removed" src="https://user-images.githubusercontent.com/8878532/190498376-9393a4b2-f4ed-48d1-a062-27158a197f3c.png"> |
|[Lower Overlay Opacity](https://trello.com/c/vNiGlWsE/29-lower-overlay-opacity) |<img width="211" alt="Overlay color image" src="https://user-images.githubusercontent.com/8878532/190500675-c0687219-96a2-47d7-a25e-528bba51cf9a.png"> |
|[Podcast Card Updates](https://trello.com/c/dTzY2dVF/20-podcast-cards) |<img width="211" alt="Updated Podcast Card" src="https://user-images.githubusercontent.com/8878532/190501073-331425f1-e547-4fe9-ad55-51e44b097709.png"> |
|[Primary Color Variables](https://trello.com/c/GvlS3oQj/26-primary-color-variables) | <img width="211" alt="Color Variables" src="https://user-images.githubusercontent.com/8878532/190501340-30e9ce3e-037c-42dc-9ca8-87c7bcc80884.png"> |
|[Report bugs button on debug screen](https://trello.com/c/RmdZZaaY/23-report-bugs-button-on-debug-screen) |<img width="211" alt="New Report Bug Button" src="https://user-images.githubusercontent.com/8878532/190502889-3494e067-aee3-406b-a5ad-d4ac046109cb.png"> |
|[Update Podcast Favorite Switch](https://trello.com/c/1repf470/24-podcast-favorite-switch) | <img width="211" alt="Updated Podcast Favorite Switch Photo" src="https://user-images.githubusercontent.com/8878532/190503611-8aef3e05-e073-408b-9056-7ed0f2140a8d.png"> |
|[Update Toggle Colors](https://trello.com/c/ALFyorCR/21-control-colors) |<img width="211" alt ="New Toggle Colors" src="https://user-images.githubusercontent.com/8878532/190503921-f93cbd0e-b4f8-4c6a-9d73-15eb03b46e57.png"> |

jamonholmgren commented on boilerplate/app/components/Toggle.tsx:
```diff
@@ -184,26 +184,29 @@ function Checkbox(props: ToggleInputProps) {
 
   const offBackgroundColor = [
-    disabled && colors.palette.neutral300,
+    disabled && colors.palette.neutral400,
```
Generally speaking, we should avoid using `colors.palette` unless it's a very one-off situation. Instead, we should have a semantic name, like for example `colors.inactiveElement` or something.

cdanwards replied:
@jamonholmgren following up on this. In a situation like this where it's used often within a component, should we create a new semantic name within this context or just use the `colors.border` that's already defined?

jamonholmgren replied:
@cdanwards I honestly am the worst person to ask ... maybe @robinheinze or @yulolimum?

robinheinze replied:
Since this is not a border, I would not use `colors.border`. If this is only used in this component and doesn't represent an intentional design choice to make all "inactive elements" this color throughout the app, then I think it's fine to use palette.

jamonholmgren commented on boilerplate/app/screens/DemoPodcastListScreen.tsx:
```diff
@@ -203,50 +210,65 @@ const EpisodeCard = observer(function EpisodeCard({
+          }
+        >
+          <Text size="xxs" accessibilityLabel={episode.duration.accessibilityLabel}>
+            {isFavorite ? "Unfavorite" : "Favorite"}
```
I think this should be translated via the `translate` function, as above.

jamonholmgren reviewed (commented):
Some minor feedback. Not sure what the CI failures are about.

jamonholmgren commented:
@cdanwards There are some Prettier errors, can you fix?

cdanwards commented:
@mazenchami @kateinkim @leonskim hey! Do y'all mind helping me update the `ar.ts` and `ko.ts` files to match? I added 2 things and changed another. Thank you!

kateinkim commented on boilerplate/app/i18n/ko.ts:
```diff
@@ -99,6 +99,8 @@ const ko: Translations = {
   demoPodcastListScreen: {
     title: "React Native 라디오 에피소드",
     onlyFavorites: "즐겨찾기만 보기",
+    favoriteButton: "Favorite",
+    unfavoriteButton: "Unfavorite",
```
```suggestion
    favoriteButton: "즐겨찾기,
    unfavoriteButton: "즐겨찾기 해제",
```

cdanwards commented on boilerplate/app/i18n/ko.ts:
```diff
@@ -82,7 +82,7 @@ const ko: Translations = {
       "축하합니다. 여기 아주 고급스러운 React Native 앱 템플릿이 있습니다. 이 보일러 플레이트를 사용해보세요!",
     reactotron: "Reactotron 으로 보내기",
-    reportBugs: "Ignite 에 관한 버그 보고하기",
+    reportBugs: "버그 보고하기",
```
@kateinkim does this look right?

kateinkim replied:
Yes! perfect! sorry If I missed this.

cdanwards replied:
No worries! Thank you!

kateinkim commented on boilerplate/app/i18n/ko.ts:
```diff
@@ -99,7 +99,7 @@ const ko: Translations = {
     title: "React Native 라디오 에피소드",
     onlyFavorites: "즐겨찾기만 보기",
-    favoriteButton: "즐겨찾기,
+    favoriteButton: "즐겨찾기",
```
Ooops!

joshuayoes reviewed (approved):
I've already been working with Dan on this, but it looks good to go for me!
